en/clubs/dkm/body.md

```diff
@@ -1 +1,597 @@
-# Under construction
+<style>
+ img {
+   width: 165px;
+   height: 165px;
+   border-radius: 50%;
+   object-fit: cover;
+   box-sizing: content-box;
+   padding-top: 10px;
+ }
+ </style>
+
+# DKM
+DKM is Datas Klubbmästeri (Pub organizers at the computer science chapter of KTH). We ensure that everyone who studies computer science at KTH can have fun from time to time and socialize with others after school hours. We arrange parties, pubs and awesome events throughout the year! If you want to come in contact with us you can do so via: [dkm@d.kth.se](mailto:dkm@d.kth.se)!
+
+## What do we do?
+
+Every Wednesday we open the doors to META (Osquars backe 21) and give you the chance to relax and enjoy alcoholic or alcohol free beverages ranging from beer and cider to freshly made cocktails. The pub opens 17:17 and usually is open until we or the guests has had enough. Take the chance to take a break from all the studying and let loose on the dance floor! 
+
+After the exam period you might want to relax that little bit extra. How fitting that DKM arranges an Exam Pub (a.k.a Tentapub), which typically takes place on the first saturday after the exam period. DKM arranges two per year and MKM (Medias Klubbmästeri) arranges the other two. These events are attract a lot of people since lots of other pubs around campus are open at the time, so be here in time! 
```

Comment:
```suggestion
After the exam period you might want to relax that little bit extra. How fitting that DKM arranges an Exam Pub (a.k.a Tentapub), which typically takes place on the first Saturday after the exam period. DKM arranges two per year and MKM (Medias Klubbmästeri) arranges the other two. These events are attract a lot of people since lots of other pubs around campus are open at the time, so be here in time! 
```